build-tools.Dockerfile

```diff
@@ -226,7 +239,7 @@ RUN wget -O /tmp/libicu-${ICU_VERSION}.tgz https://github.com/unicode-org/icu/re
 USER nonroot:nonroot
 WORKDIR /home/nonroot
 
-RUN echo -e "--retry-connrefused\n--connect-timeout 15\n--retry 5\n--max-time 300\n" > /home/nonroot/.curlrc
+RUN echo "--retry-connrefused\n--connect-timeout 15\n--retry 5\n--max-time 300\n" > /home/nonroot/.curlrc
```

Comment:
Hmm, aren't we are using 'bash' here? There's a `SHELL ["/bin/bash", "-euo", "pipefail", "-c"]`command on line 68

Reply:
yes, I missed that part... and now I think that it's better idea to set `SHELL` to `bash` everywhere :)